projects/components/src/form/form-select/form-select.component.spec.ts

```diff
@@ -105,3 +112,8 @@ class TestHostComponent {
         });
     }
 }
+
+const optionWithValueAsNumber: SelectOption = {
```

Comment:
Please create a function that returns a new object every time instead of passing an object that is likely to be shared between tests